Version_Control_Exercise.md

```diff
@@ -0,0 +1,183 @@
+# Version Control Exercise
+Note: This tutorial is based on the EECS 280 Git Tutorial from Lab 0. All credit goes to James Juett and Zachary Goldston for designing the original lab.
+
+Before beginning this exercise, we recommend going through the main README in the `workflow-intros` repository, to get some basic exposure to git. 
+
+For the purposes of these exercises, we recommend you work in groups of 2-3 to go through this tutorial. This way, you and your group will gain a basic understanding of Git Workflow and handling code changes across different remotes/computers. For the purposes of this tutorial, we will assume a group of 3, but feel free to adjust roles to the number of people in your group.
+
+## Initial Steps
+
+All group members should verify `git` is properly installed via the command `git --version` in your terminal. Additionally, all group members should configure their git profile, if not done yet. You can do this by running the two commands:
+
+    git config --global user.name "username"
+    git config --global user.email "youremail"
+
+(Replace the `username` and `youremail` parts with the GitHub username and email you plan to use when working on MRover! Note that we do not use Michigan's GitHub server)
+
+Everyone in the group should sign into their GitHub account they plan to use with the MRover team, if they have not done already.
+
+**Person 1** should create a new repository on GitHub.  From here, Person 1 should ensure the following when creating the new project:
+* Name it `mrover-vc-exercise`
+* Set it to private.
+* Select the "Add a README file" option. Make sure it is checked before creating.
+
+Once **Person 1** has created the repository, the next step is to add the other members of the group. Click on the three dots towards the right of the newly created repository page, select "Settings," then "Collaborators," and then "Add people." You will want to add them via username, assuming everyone in the group has made a GitHub account. Once done, you all have access to your **remote repository**, where you can put your exercise code that lives "in the cloud."
+
+To actually work on the code, you will need to **clone** the remote repository to create a **local repository** on your own computer.
+ 1. Open a terminal and navigate to a folder. We recommend making one called `mrover` to store all the work you will be doing with us.
+ 2. Run the following command (replace `zwgold` with the username of the person who had created the remote repository): `git clone git@github.com:zwgold/mrover-vc-exercise.git` (Note: This command will only work if you have SSH Keys set up. Please follow the steps in the first tutorial to do so!)
```

Comment:
Could this be changed to describe how to find the link from the browser page? Personally, I very rarely just type the link directly, usually I copy it from the 'code' drop down